source/docs/advanced/payments/adyen.md

```diff
@@ -5,46 +5,67 @@ title: Adyen integration
+- `FRONT_COMMERCE_ADYEN_API_KEY` The API key. You can find it in your adyen Customer Area under `Developers > API Credential > [your_prefered_credential] > API Key > API Key`
+- `FRONT_COMMERCE_ADYEN_LIVE_URL_PREFIX` only needed in production environment. Should be configured to contain [the adyen live URL prefix](https://docs.adyen.com/development-resources/live-endpoints#live-url-prefix)
+- `FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME` (more on [webhooks below](#Add-webhook)) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > User Name`
+- `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` (more on [webhooks below)](#Add-webhook) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > Password`
+- `FRONT_COMMERCE_ADYEN_HMAC_KEY` (more on [webhooks below](#Add-webhook)) create it from the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Additional Settings > HMAC key`
+- `FRONT_COMMERCE_ADYEN_PREVIOUS_HMAC_KEY` (more on [webhooks below](#Add-webhook)) when you regenerate your HMAC key configure this to be the old one
 
 ```sh
 # In .env
+FRONT_COMMERCE_ADYEN_MERCHANT_ACCOUNT=ExampleMerchant
 FRONT_COMMERCE_ADYEN_CLIENT_KEY=live_32charactersstring
+FRONT_COMMERCE_ADYEN_API_KEY=a_very_very_very_long_key
+FRONT_COMMERCE_ADYEN_LIVE_URL_PREFIX=1797a841fbb37ca7-AdyenDemo-
+FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME=a_username
+FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD=a_password
+FRONT_COMMERCE_ADYEN_HMAC_KEY=the_hmac_key
+FRONT_COMMERCE_ADYEN_PREVIOUS_HMAC_KEY=the_previous_hmac_key
 # the Adyen client key is prefixed with live_ or test_
 ```
 
-### Register the Adyen for Magento2 payment module in Front-Commerce
+### Register the Adyen payment module in Front-Commerce
 
 In your Front-Commerce application:
 
+For magento1:
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/payment-adyen"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+-    { name: "Magento1", path: "server/modules/magento1" }
++    { name: "Magento1", path: "server/modules/magento1" },
++    {
++      name: "Magento1Adyen",
++      path: "payment-adyen/server/modules/payment-adyen",
++    }
+   ],
+   webModules: [
+-    { name: "FrontCommerce", path: "front-commerce/src/web" }
++    { name: "FrontCommerce", path: "front-commerce/src/web" },
++    {
++      name: "Magento1Adyen",
```

Comment:
this part is not specific to Magento1, to limit possible confusion, it could be called just `Adyen` and same for the Magento2 case